static/locales/en-US.yaml

```
@@ -871,6 +870,20 @@ Playlist:
   View: View
   Views: Views
   Last Updated On: Last Updated On
+  Sort By:
+    Sort By: Sort By
+    Custom: Custom
+    CustomDescending: Custom (descending)
+    Default: Default
+    DefaultDescending: Default (descending)
+    DateAddedNewest: Date added (newest)
+    DateAddedOldest: Date added (oldest)
+    DatePublishedNewest: Date published (newest)
+    DatePublishedOldest: Date published (oldest)
```

Also it's not date but time but I am fine with the key
```suggestion
    DateAddedNewest: Latest added first
    DateAddedOldest: Earliest added first
    DatePublishedNewest: Latest published first
    DatePublishedOldest: Earliest published first
```